.github/workflows/ci.yml

```diff
@@ -102,6 +102,13 @@ jobs:
           # do not release iso image until issue is fixed
           # see: https://github.com/dasharo/dasharo-issues/issues/288
           # scp -i ~/.ssh/dts-ci-key dts-base-image-${DTS_VER}.iso builder@10.1.40.2:boot/dts/${DTS_VER}/
+      - name: Deploy manifest on boot.dasharo.com
+        shell: bash
+        run: |
+          DTS_VER="${{steps.dts-ver.outputs.DTS_VER}}"
```

Comment:
Everywhere else it's `DTS_VER="${{ env.DTS_VER }}"`, maybe keep it the same unless there is a reason?

Reply:
Good catch 0b857c9620c1f78e2b4fa648fe7f2011c39307b6